api/nodes/filters.py

```diff
@@ -25,7 +25,7 @@ def _operation_to_query(self, operation):
         return super(NodesListFilterMixin, self)._operation_to_query(operation)
 
 
-class NodePreprintsFilterMixin(ODMFilterMixin):
+class NodeODMFilterMixin(ODMFilterMixin):
```

Comment:
Because NodeForksList (etc) having NodePreprintsFilterMixin does not make sense. Imho it is a generic mixin that happens to contain preprint specific behavior. In addition, I'm not 100% sure it actually is required anymore but left it in as a precaution. Seemed silly to me.